jetty-webapp/src/main/java/org/eclipse/jetty/webapp/WebAppClassLoader.java

```diff
@@ -644,8 +643,18 @@ public boolean removeTransformer(ClassFileTransformer transformer)
         
         return webapp_class;
     }
-    
-    
+
+    /**
+     * Get the URL by class name.
+     * @param name the name of the class to load
+     * @return the class URL
+     */
+    protected URL getClassUrl(final String name)
```

Comment:
This isn't a URL, it's more of a resource string reference.

Taking `org.eclipse.jetty.util.log.StdErrLog` and making it `org/eclipse/jetty/util/log/StdErrLog.class`

Reply:
Is it ok, if I will rename the method to the `getResourceClassReference`?